packages/bpk-component-modal/readme.md

```diff
@@ -84,12 +85,15 @@ class App extends Component {
 | getApplicationElement | func                 | true     | -             |
 | closeLabel            | string               | false    | null          |
 | closeText             | string               | false    | null          |
-| renderTarget          | instance of Element  | false    | null          |
+| renderTarget          | func                 | false    | null          |
 | wide                  | bool                 | false    | false         |
 
 ## Theme Props
 
-Both all the props from [link](/components/web/typography#links) and
-
-* `modalCloseButtonColor`
-* `modalCloseButtonHoverColor`
```

Comment:
so no links?  the nudger is using a link to the secondary button, should we update that one as well?

Reply:
Ah ok. I removed it because the sentence was unfinished. I guess its awkward here because we don't have a docs link for the close button component 🙈

Reply:
You mean here https://backpack.github.io/components/web/nudgers/#theme-props. 

I see. I guess this is a potential decision™ we need to make. There are pros and cons - indirection sucks for consumer but it's nice as a maintainer to only have to update one place.

Reply:
too many decisions for one day!

Reply:
as discussed offline we'll keep the list of all the the attribute explicit so it'll be easier to consume